server/cli/accounting.go

```diff
@@ -48,37 +48,39 @@ func runGetAcct(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	server.RunGetAcct(Context, args[0])
+	client := client.NewAdminClient(&Context.Context, Context.Addr, client.Accounting)
+	body, err := client.GetYAML(args[0])
+	if err != nil {
+		log.Error(err)
+		return
+	}
+	fmt.Printf("Accounting config for prefix %q:\n%s\n", args[0], body)
 }
 
 // A lsAcctsCmd command displays a list of acct configs by prefix.
 var lsAcctsCmd = &cobra.Command{
-	Use:   "ls [options] [key-regexp]",
```

Comment:
I've intentionally removed the key regexp. This was client side only and only necessary if we reach very large number of configs. I'd say let's bring it back if it's really needed.